Week2/app.js

```diff
@@ -0,0 +1,92 @@
+const mysql = require('mysql');
+const prompt = require('prompt');
+const util = require('util');
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  password: 'password',
+  database: 'new_world',
+  multipleStatements: false,
+});
+
+const execQuery = util.promisify(connection.query.bind(connection));
+const input = util.promisify(prompt.get.bind(this));
+
+async function queryDb() {
+  let userMakeChoices = '';
+  prompt.start();
+
+  try {
+    console.log(`Please choose one of the questions below(type the numbers from 1 to 5):
+
+    1. What is the capital of country X ? (Accept X from user)
+    2. List all the languages spoken in the region Y (Accept Y from user)
+    3. Find the number of cities in which language Z is spoken (Accept Z from user)
+    4.Accept the region and language from the user.
+    5. List all the continents with the number of languages spoken in each continent
+    `);
+
+    const choices = await input(['choice']);
+    userMakeChoices = choices.choice;
+
+    const chooseFirstQuery = `SELECT name from city where id in (select capital from country where name = ?)`;
+    const chooseSecondQuery = `SELECT countrylanguage.language ,country.region FROM country JOIN countrylanguage ON countrylanguage.CountryCode = country.code WHERE country.region = ?;`;
```

Comment:
If you don't want to have duplicates you can also add a distinct here

Reply:
Thank you for the valuable feedback, i will use distinct.

Reply:
dear @mogaleaf I added the last part which is about ERD.